lib/oci_utils/impl/ocid-main.py

```diff
@@ -144,43 +150,16 @@ def getName(self):
         """
         return self.thread_name
 
-    def stop(self):
-        """
-        Tell the thread to stop.
-
-        Returns
-        -------
-            No return value.
-        """
-        # de-activate
+    def requestStop(self):
+        self.thr_logger.debug('been requested to stop')
         self.active = False
+        self.blockingEvt.set()
 
-    def get_context(self):
-        """
-        Collect the context of the thread.
-
-        Returns
-        -------
-        dict
-            The context.
-        """
-        # make sure it's not running
-        self.threadLock.acquire()
-        # save context
-        ctx = self.context
-        self.threadLock.release()
-        return ctx
-
-    def is_first_iter_done(self):
+    def waitFirstIteration(self):
```

Comment:
Why is this not wait_first_iteration, ie. standard python naming conventions?

Reply:
done